packages/api/test/__fixtures__/definitions/operationid-quirks.json

```diff
@@ -13,7 +13,7 @@
     "/quirky-operationId": {
       "get": {
         "description": "This mess of a string is intentionally nasty so we can be sure that we're not including anything that wouldn't look right as an operationID for a potential method accessor in `api`.",
-        "operationId": "find/?*!@#$%^&*()-=_.,<>+[]{}\\|pets-by_status"
+        "operationId": "quirky/?*!@#$%^&*()-=_.,<>+[]{}\\|operation-id_string"
```

Comment:
Changed this to be different because the `findPetsByStatus` operationID on a `/quirky-operationId` path was confusing the hell out of me.